MaterialDesignThemes.Wpf/DrawerOpenedEventHandler.cs

```diff
@@ -0,0 +1,4 @@
+﻿namespace MaterialDesignThemes.Wpf
+{
+    public delegate void DrawerOpenedEventHandler(object sender, DrawerOpenedEventArgs eventArgs);
```

Comment:
Rather than creating your own delegate we can simply use `EventHandler<DrawerOpenedEventArgs>`